src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -6931,6 +6976,32 @@ components:
           items:
             type: string
 
+    EnumerateSnapshotAccessRequest:
+      type: array
+      items:
+        $ref: '#/components/schemas/EnumerateSnapshotAccessRequestItem'
```

Comment:
I'm not sure how Data Explorer expects this endpoint to evolve in the future, but looking at TDR's snapshot and dataset enumeration examples could give you an idea, e.g. `EnumerateSnapshotModel`.

As a way to future-proof your response format, I'd recommend that you format your response as an object and nest the items within it like so:
```
    EnumerateSnapshotAccessRequest:
      type: object
      properties:
        items:
          type: array
          items:
            $ref: '#/components/schemas/EnumerateSnapshotAccessRequestItem'
```

That way if you want to support pagination or filtration down the line, it won't be a breaking change to add in other supporting fields in the response.

Reply:
Yes makes sense!